seed/seed.json

```diff
@@ -57,6 +57,34 @@
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
         },
+        {
+            "name": "CookieListDefaultStudy",
+            "experiments": [
+                {
+                    "name": "Enabled",
+                    "probability_weight": 100,
+                    "feature_assocatiation": {
+                        "enable_feature": ["BraveAdblockCookieListDefault"]
+                    }
+                },
+                {
+                    "name": "Disabled",
+                    "probability_weight": 100,
```

Comment:
Same as on https://github.com/brave/brave-variations/pull/188, this should be 0 right?